modules/packages/LinearAlgebra.chpl

```diff
@@ -1900,6 +1900,20 @@ module Sparse {
     return S;
   }
 
+  proc _array.plus(A: [?Adom] ?eltType) where isSparseArr(this) && !isCSArr(this)
+                                              && isSparseArr(A) && !isCSArr(A) {
```

Comment:
I think the generic `plus` and `minus` implementations can be simpler updates to the existing version, but I am not too opposed to keeping the generic ones completely separate.